samples/legacy_support/All_ID_Mapping_Original.csv

```diff
@@ -2148,91 +2148,91 @@ id,Name,LegacyKey,MainIcon,Modifier1,Modifier2,ExtraIcon,FullFrame,GeometryType,
 2154,Military (Sea Subsurface) : Submarine,S-U-S-----,35110100,,,,,Point,,,
-2155,Military (Sea Subsurface) : Submarine : Submarine Confidence : Possible Submarine Low 1,S-U-S1----,S-U-S1----,,,,,Point,,,
-2156,Military (Sea Subsurface) : Submarine : Submarine Confidence : Possible Submarine Low 2,S-U-S2----,S-U-S2----,,,,,Point,,,
-2157,Military (Sea Subsurface) : Submarine : Submarine Confidence : Possible Submarine High 3,S-U-S3----,S-U-S3----,,,,,Point,,,
-2158,Military (Sea Subsurface) : Submarine : Submarine Confidence : Possible Submarine High 4,S-U-S4----,S-U-S4----,,,,,Point,,,
-2159,Military (Sea Subsurface) : Submarine : Submarine Confidence : Probable Submarine,S-U-SL----,S-U-SL----,,,,,Point,,,
+2155,Military (Sea Subsurface) : Submarine : Submarine Confidence : Possible Submarine Low 1,S-U-S1----,S-U-S1----,,,,,Point,C,,
+2156,Military (Sea Subsurface) : Submarine : Submarine Confidence : Possible Submarine Low 2,S-U-S2----,S-U-S2----,,,,,Point,C,,
+2157,Military (Sea Subsurface) : Submarine : Submarine Confidence : Possible Submarine High 3,S-U-S3----,S-U-S3----,,,,,Point,C,,
+2158,Military (Sea Subsurface) : Submarine : Submarine Confidence : Possible Submarine High 4,S-U-S4----,S-U-S4----,,,,,Point,C,,
+2159,Military (Sea Subsurface) : Submarine : Submarine Confidence : Probable Submarine,S-U-SL----,S-U-SL----,,,,,Point,C,,
 2160,Military (Sea Subsurface) : Submarine : Submarine-Surfaced,S-U-SF----,35110101,,,,,Point,,,
-2161,Military (Sea Subsurface) : Submarine : Submarine-Surfaced : Submarine Confidence : Certain Submarine,S-U-SR----,S-U-SR----,,,,,Point,,,
-2162,Military (Sea Subsurface) : Submarine : Submarine-Bottomed,S-U-SB----,35110103,,,,,Point,,,
+2161,Military (Sea Subsurface) : Submarine : Submarine-Surfaced : Submarine Confidence : Certain Submarine,S-U-SR----,S-U-SR----,,,,,Point,C,,
+2162,Military (Sea Subsurface) : Submarine : Submarine-Bottomed,S-U-SB----,35110103,,,,,Point,C,,
 2163,Military (Sea Subsurface) : Other Submersible : Original,S-U-SO----,S-U-SO----,,,,,Point,,,
-2164,Military (Sea Subsurface) : Nonsubmarine : Original,S-U-SX----,S-U-SX----,,,,,Point,,,
+2164,Military (Sea Subsurface) : Nonsubmarine : Original,S-U-SX----,S-U-SX----,,,,,Point,C,,
 2165,Military (Sea Subsurface) : Autonomous Underwater Vehicle (AUV),S-U-SU----,35110400,,,,,Point,,,
 2166,Military (Sea Subsurface) : Autonomous Underwater Vehicle (AUV) : Mine Warfare,S-U-SUM---,S-U-SUM---,,,,,Point,,,
 2167,Military (Sea Subsurface) : Autonomous Underwater Vehicle (AUV) : Mission Area : Antisubmarine Warfare (Sea Subsurface),S-U-SUS---,35110400,35011,,,,Point,,,
 2168,Civilian (Sea Subsurface) : Diver,S-U-ND----,35120300,,,,,Point,,,
 2169,Weapon (Sea Subsurface),S-U-W-----,35130000,,,,,Point,,,
 2170,Weapon (Sea Subsurface) : Torpedo,S-U-WT----,35130100,,,,,Point,,,
-2171,Weapon (Sea Subsurface) : Decoy,S-U-WD----,35130300,,,,,Point,,,
+2171,Weapon (Sea Subsurface) : Decoy,S-U-WD----,S-S-CD----,,,,,Point,,,
```

Comment:
@csmoore : Referencing one symbol set from another is not recommended.  I'd suggest creating a copy of S-S-CD----.svg and naming it S-U-WD----.svg.